documentation/asciidoc/computers/config_txt/boot.adoc

```diff
@@ -109,6 +109,12 @@ The `boot_delay` command instructs to wait for a given number of seconds in `sta
 
 The `boot_delay_ms` command means wait for a given number of milliseconds in `start.elf`, together with `boot_delay`, before loading the kernel. The default value is `0`.
 
+
+[[disable_poe_fan]]
+=== `disable_poe_fan`
+
+Set this option to `1` to prevent control of the PoE HAT fan through I2C (on pins ID_SD & ID_SC). Without this, a probe on the I2C bus will happen at startup, even when not using the PoE HAT. Useful when trying to minimise boot time.
```

Comment:
Perhaps better as, "By default, a probe on the I2C bus will happen at startup, even when a PoE HAT is not attached. Setting this option to `1` disables control of a PoE HAT fan through I2C (on pins ID_SD & ID_SC). If you are not intending to use a PoE HAT doing this is useful if you need to minimise boot time."

The rephrase puts the peril and the solution in better narrative order.